packages/shared/src/components/modals/streaks/NewStreakModal.tsx

```diff
@@ -12,11 +12,12 @@ const Paragraph = classed('p', 'text-center text-theme-label-tertiary');
 
 export default function NewStreakModal({
   currentStreak,
+  maxStreak,
   onRequestClose,
   ...props
 }: StreakModalProps): ReactElement {
   const { toggleOptOutWeeklyGoal, optOutWeeklyGoal } = useSettingsContext();
-  const shouldShowSplash = currentStreak > 20;
+  const shouldShowSplash = currentStreak >= maxStreak;
```

Comment:
Shouldn't it be `>` instead of `>=`? If your max streak was 20, and your new current streak is 20, you haven't really beaten a record?

Reply:
We generally update both values at the same time. So maxStreak is usually the same as currentStreak if you are beating it. We could use `===` as well here, but `>=` is one less character to write 😅

Reply:
See related PR
- https://github.com/dailydotdev/daily-api/pull/1726